README.md

```diff
@@ -372,25 +371,6 @@ end
 admin_client = Octokit.enterprise_admin_client.new
 ```
 
-### Interacting with the GitHub Enterprise Management Console APIs
-
-The GitHub Enterprise Management Console APIs are also under a separate client: `EnterpriseManagementConsoleClient`. In order to use it, you'll need to provide both your management console password as well as the endpoint to your management console. This is different from the API endpoint provided above.
-
-```ruby
-management_console_client = Octokit::EnterpriseManagementConsoleClient.new(
-  :management_console_password => "secret",
-  :management_console_endpoint = "https://hostname:8633"
-)
-
-# or
-Octokit.configure do |c|
-  c.management_console_endpoint = "https://hostname:8633"
-  c.management_console_password = "secret"
-end
-
-management_console_client = Octokit.enterprise_management_console_client.new
-```
-
```

Comment:
Seems like we didn't add that for the GHES Manage API back when we shipped its client here.
What do you think about rewriting this section for the new API?